packages/devtools_app/lib/src/scaffold.dart

```diff
@@ -301,15 +302,19 @@ class DevToolsScaffoldState extends State<DevToolsScaffold>
         create: (_) => BannerMessagesController(),
         child: DragAndDrop(
           handleDrop: _importController.importData,
-          child: Scaffold(
-            appBar: widget.embed ? null : _buildAppBar(),
-            body: TabBarView(
-              physics: defaultTabBarViewPhysics,
-              controller: _tabController,
-              children: tabBodies,
+          child: Title(
+            title: title,
+            color: Colors.blue,
```

Comment:
Where is this blue color? can you add a screenshot to the PR description?

Reply:
The blue color is a hack to make this prototype work. The color only matters on android. Switched to mirror what WidgetsApp does and added a comment.

Btw,  the title matters to change the title displayed in the web browser tab.